spec/features/code_of_conduct_spec.rb

```diff
@@ -62,5 +61,34 @@
         end
       end
     end
+
+    describe 'as a participant' do
+      let!(:organization) { create(:organization, code_of_conduct: sample_text) }
+      let!(:participant) { create(:user) }
+
+      before { sign_in participant }
+
+      it 'must be accepted', js: true do
+        visit conferences_path
+        within "#conference-#{conference.id}" do
+          click_on 'Register'
+        end
+        expect(page).to have_text('I have read and accept the Code of Conduct')
+        expect(page).not_to have_text(sample_text)
+        within 'form' do
+          click_on 'Code of Conduct'
+        end
+        expect(page).to have_content(sample_text)
+        find('button.close').click
+        click_on 'Register'
+        expect(conference.user_registered?(participant)).to be(false)
+        expect(page).to have_content('Accepted code of conduct must be accepted')
+        check 'registration[accepted_code_of_conduct]'
+        click_on 'Register'
+        expect(conference.user_registered?(participant)).to be(true)
```

Comment:
I would use the `be_true` matcher instead: `to be_true` instead of `to be(true)`